### PR TITLE
Editorial: Change exceptions to be more precise

### LINKED ISSUE #63: Be more precise about exceptions
When reviewing I noticed multiple places that did something along the lines of:

> `throw an {{InvalidStateError}}`

That should really be:

> `throw an "{{InvalidStateError}}" {{DOMException}}`

### PR DESCRIPTION
**Pull request by a-sully**
Fixes #63

Before: {{InvalidStateError Error}}

After: "{{InvalidStateError Error}}" {{DOMException}}

***
<a href="https://whatpr.org/fs/70.html" title="Last updated on Nov 2, 2022, 7:49 AM UTC (4390bf5)">Preview</a> | <a href="https://whatpr.org/fs/70/abad896...4390bf5.html" title="Last updated on Nov 2, 2022, 7:49 AM UTC (4390bf5)">Diff</a>

### CODE REVIEW

**Comment by annevk** [comment]
This seems to include some of the other changes. Should we work on those first and then rebase this?

**Comment by a-sully** [comment]
Whoops, sorry about that. I hadn't meant to include those other changes here. There's some merge conflicts so I'll rebase once the other changes merge

File: index.bs [hunk 1]
```diff
@@ -654,13 +654,13 @@ The <dfn method for=FileSystemDirectoryHandle>removeEntry(|name|, |options|)</df
      [=entry/request access=] given "`readwrite`".
      If that throws an exception, [=reject=] |result| with that exception and abort.
   1. If |access| is not "{{PermissionState/granted}}",
-     reject |result| with a {{NotAllowedError}} and abort.
+     reject |result| with a "{{NotAllowedError}}"" {{DOMException}} and abort.
```
Comment (annevk) [inline index.bs:657]:
```suggestion
     reject |result| with a "{{NotAllowedError}}" {{DOMException}} and abort.
```

**Review by annevk — APPROVED** [review]

**Comment by annevk** [comment]
It seems this has been impacted by https://github.com/whatwg/whatwg.org/issues/401. Will try to sort that, but not sure about the timeline.

**Comment by annevk** [comment]
By the way, you want stuff like "Fixes #63" to be in the commit message body so you don't have to visit the PR to find the issue discussion.

**Pull request merged**